tests/clar/fs.h

```diff
@@ -273,10 +273,16 @@ cl_fs_cleanup(void)
 # include <sys/sendfile.h>
 #endif
 
-#if defined(__APPLE__) || defined(__FreeBSD__)
+#if defined(__APPLE__)
 # include <copyfile.h>
 #endif
 
+#if defined(__FreeBSD__)
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <sys/uio.h>
+#endif
```

Comment:
I don't think that this is necessary.  `sys/types.h` is already included (above) and none of these changes appear to be necessary on my FreeBSD 12 system.